src/aws_sagemaker/aws_sagemaker.py

```diff
@@ -0,0 +1,229 @@
+                                 count: int,
+                                 *, 
+                                 extra: Optional[Dict[str, Any]] = None,
+                                 timeout: Optional[float] = None,
+                                 **kwargs) -> List[Classification]:
+        classifications = []
+        if isinstance(image, RawImage):
+            response = self.client.invoke_endpoint(EndpointName=self.endpoint_name, 
+                                                   ContentType= 'application/x-image',
+                                                   Accept='application/json;verbose',
+                                                   Body=image.data) # send image.data[24:]??
+        else:
+            stream = BytesIO()
+            image = image.convert("RGB")
+            image.save(stream, "JPEG")
+            response = self.client.invoke_endpoint(EndpointName=self.endpoint_name, 
+                                                   ContentType= 'application/x-image',
+                                                   Accept='application/json;verbose',
+                                                   Body=stream.getvalue())
+            
+        # Either way... 
+        out = json.loads(response['Body'].read())
+        labels = out['labels']
+        probs = out['probabilities']
+        zipped = list(zip(labels, probs)) 
+        res = sorted(zipped, key = lambda x: -x[1]) # zipped in decreasing probability order
+        for i in range(count):
+            classifications.append({"class_name": res[i][0], "confidence": res[i][1]})
+
+        return classifications
+    
+
+    async def get_classifications_from_camera(self, 
+                                              camera_name: str, 
+                                              count: int, 
+                                              *,
+                                              extra: Optional[Dict[str, Any]] = None,
+                                              timeout: Optional[float] = None,
+                                              **kwargs) -> List[Classification]:
+        if camera_name != self.camera_name:
+            raise Exception(
+                "Camera name given to method",camera_name, " is not the one given to configuration ",self.camera_name)
+        cam = self.camera
+        img = await cam.get_image()
+        return await self.get_classifications(image=img, count=count)
+
+    
+    async def get_detections(self,
+                            image: Union[Image.Image, RawImage],
+                            *,
+                            extra: Optional[Dict[str, Any]] = None,
+                            timeout: Optional[float] = None,
+                            **kwargs) -> List[Detection]:
+        
+        detections = []
+        if isinstance(image, RawImage):
+            width, height = int.from_bytes(image.data[8:16], "big"), int.from_bytes(image.data[16:24], "big")
```

Comment:
before you do this, you have to check the mime_type field of the RawImage to know if you are dealing with a vnd.viam.rgba or another kind of image type.